### PR TITLE
[ENH] add frontend config

### LINKED ISSUE #228: build meta analysis creation page

### PR DESCRIPTION
**Pull request by jdkent**
add configuration for frontend to create options for specifying meta-analyses.

each parameter now has: "type", "description", and "default", so you can prefill certain options with the default value. In addition, each Estimator/class has a "Summary" so you can give a hoverable/inline description of the choices the user has when selecting. Finally, I added CORRECTOR and populated IBMA with algorithms, so that Algorithm and Corrector selection can be completed.

- current drawbacks:
  - does not include optional arguments of base classes (e.g., masker, resample_kwargs)
  - some string arguments are actuallly references to files (need file upload feature?)

- [ ] replace single quotes with double quotes in the config (invalid json) @jdkent

### CODE REVIEW

File: synth/neurosynth-frontend/scripts/parse_nimare_docs.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "parameters": {
+            param.arg_name: {
+                "description": param.description,
+                "type": param.type_name,
+            } for param in docs.params if param.arg_name != '**kwargs'
+        }
+    }
+
+# save config file
+fname = Path(__file__).parent.parent / "src" / "assets" / "config" / "config_auto.json"
```
Comment (adelavega) [inline synth/neurosynth-frontend/scripts/parse_nimare_docs.py:54]: maybe give this a better name

**Review by adelavega — CHANGES_REQUESTED** [review]
Looks great! One minor comment

File: synth/neurosynth-frontend/scripts/parse_nimare_docs.py (hunk 1 above)
Comment (jdkent) [reply synth/neurosynth-frontend/scripts/parse_nimare_docs.py]: changed to `meta_analysis_params.json`

File: synth/neurosynth-frontend/src/assets/config/meta_analysis_params.json [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+                    "default": 0.001
+                },
+                "n_iters": {
+                    "description": "Number of iterations for correction. Default: 10000",
+                    "type": "int",
+                    "default": 10000
+                },
+                "xyz": {
+                    "description": "Tab-delimited file of coordinates from database or numpy array with XYZ coordinates. Voxels are rows and x, y, z (meaning coordinates) values are the three columnns.",
+                    "type": "str",
```
Comment (jdkent) [inline synth/neurosynth-frontend/src/assets/config/meta_analysis_params.json:159]: this will likely need to be a file upload (not just a string).

**Pull request merged**